app/views/stats/_range.html.erb

```diff
@@ -183,7 +183,7 @@ master
 </div>
 <script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
 <script>
-var questions = [<%= @questions.count %>];
+var questions = [<%= @questions %>];
```

Comment:
Hi @cesswairimu can you check my work here? These tests had been silently failing, they're not too bad, just trying to resolve them all. Thanks!

Reply:
Sorry I missed this, Taking a look now,